src/test/java/org/symphonyoss/symphony/messageml/elements/form/DateSelectorTest.java

```diff
@@ -9,6 +9,7 @@
 import org.symphonyoss.symphony.messageml.exceptions.InvalidInputException;
 
 import static org.junit.Assert.assertEquals;
+import static org.symphonyoss.symphony.messageml.markdown.MarkdownRenderer.addEscapeCharacter;
```

Comment:
My concern is that you test the output by constructing the expected result by using the same code that you want test 😄 

It is not a big issue (as the code is simple) but maybe add also a JUnit for addEscapeCharacter then...

Reply:
Is not used in every test, in some of them for example TextArea, Radio and DatePicker I wrote directly the expected markdown. I tested with _ and - characters, should be enough WDYT?